### PR TITLE
Add `Lazer Grand Arena: Registrations now open!` wiki article and news post

### PR DESCRIPTION
**Pull request by LeoFLT**
Don't mind the truckload of commits

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+THE LAZER GRAND ARENA!
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-06-18-lga-announcement.mp4?v=0" type="video/mp4" preload="none">
+    </video>
+</div>
+
+This brand new event will pit the world's 32 best players against eachother in a 1v1 played on **osu!(lazer)!**
```
Comment (pishifat) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:19]:
```suggestion
This brand new event will pit the world's 32 best players against each other in a 1v1 played on **osu!(lazer)!**
```

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Lazer Grand Arena: Registrations now open!"
+date: 2024-06-18 10:00:00 +0000
+---
+
+While it's great to see countries rallying together and forming teams to compete, sometimes we have to wonder what players could truly do if they didn't have to pick around their teammates and truly shine on their own, and with the recent improvements made to osu!'s future client, we feel like it's finally time to bring to you...
```
Comment (pishifat) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:7]: double "truly' sounds kinda weird

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-06-18-lga-announcement.mp4?v=0" type="video/mp4" preload="none">
+    </video>
+</div>
+
+This brand new event will pit the world's 32 best players against eachother in a 1v1 played on **osu!(lazer)!**
+
+The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle eachother using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```
Comment (pishifat) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:21]:
```suggestion
The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle each other using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md (hunk 1 above)
Comment (Walavouchey) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:19]:
```suggestion
This brand-new event will pit the world's 32 best players against each other in 1v1 matches played on **osu!(lazer)!**
```

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md (hunk 3 above)
Comment (Walavouchey) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:21]:
```suggestion
The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full Featured Artist lineup, the players will now battle each other using **cumulative score difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+date: 2024-06-18 10:00:00 +0000
+---
+
+While it's great to see countries rallying together and forming teams to compete, sometimes we have to wonder what players could truly do if they didn't have to pick around their teammates and truly shine on their own, and with the recent improvements made to osu!'s future client, we feel like it's finally time to bring to you...
+
+![](/wiki/Tournaments/LGA/2024/img/lga2024-banner.jpg)
+
+THE LAZER GRAND ARENA!
+
+<div align="center">
```
Comment (Walavouchey) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:13]: fixes smol spacing

image


```suggestion
<div align="center" class="osu-md__paragraph">
```

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+This brand new event will pit the world's 32 best players against eachother in a 1v1 played on **osu!(lazer)!**
+
+The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle eachother using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | June 18 (10:00 UTC) to July 01 (23:59 UTC) |
+| Qualifier and tournament format showcase | July 06 (14:00 UTC) |
```
Comment (Walavouchey) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:28]: imo

```suggestion
| Registration phase | June 18 (10:00 UTC) to July 1 (23:59 UTC) |
| Qualifier and tournament format showcase | July 6 (14:00 UTC) |
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2024-06-18 (10:00 UTC)/2024-07-01 (23:59 UTC) |
+| Qualifier and tournament format showcase | 2024-07-07 (14:00 UTC) |
+| Qualifier stage | 2024-07-13/2024-07-14 |
+| Bracket stage (single elimination) | 2024-07-20/2024-07-21 |
+| Top 8 (single elimination)/Top 4 (double elimination) | 2024-07-27/2024-07-28 |
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:25]: sentence case

```suggestion
| Top 8 (single elimination)/top 4 (double elimination) | 2024-07-27/2024-07-28 |
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+## Prizes
+
+The minimum prize pool will be revealed after the Qualifier stage.
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | profile badge, %50 of the prize pool, *more prizes TBA* |
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:33]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, %50 of the prize pool, *more prizes TBA* |
```

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md (hunk 2 above)
Comment (LeoFLT) [reply news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md]: tried a fix on 0e3d500

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md (hunk 3 above)
Comment (LeoFLT) [reply news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md]: applied similar reviews from wala (ty for the quick review though)

File: wiki/Tournaments/LGA/2024/en.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+14. Referees may allow, at their discretion, lower or higher tolerances for timers.
+15. The tournament managers may request liveplays or recordings of individual players at any point in the tournament without prior warning.
+16. The tournament managers will be responsible for receiving and investigating any tournament-related complaints.
+17. The tournament managers reserve the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Tournament registration
+
+1. Every user interested in will be required to signs up by utilising the osu! tournament listing [linked above](#links).
+2. To ensure valid registrations, every prospective participant will be manually checked by the [account support team](/wiki/People/Account_support_team), in a manner similar to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening) that is offered to community tournaments. To be successfully accepted on the list, players are required to be placed #2500 or higher on the osu! global ranking, and not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:97]:
```suggestion
2. To ensure valid registrations, every prospective participant will be manually checked by the [account support team](/wiki/People/Account_support_team), in a manner similar to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening) that is offered to community tournaments. To be successfully accepted on the list, players are required to be placed #2,500 or higher on the osu! global ranking, and not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+1. In the Qualifier stage, all players will play a specific pool designed by the mappool selectors.
+2. The mappool will contain 4 brackets: No Mod, [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), and [Double Time](/wiki/Gameplay/Game_modifier/Double_Time).
+3. The Qualifier pool will feature 11 beatmaps, organised as such:
+   - No Mod: 4 beatmaps
+   - Hidden: 2 beatmaps
+   - Hard Rock: 2 beatmaps
+   - Double Time: 3 beatmaps
+4. Players will be asked to play the mappool twice at a designated time. The best play for each map will be used for seeding.
+5. The mappool is to be played according to the order listed on this page.
+6. Only the top 32 seeded players will advance to the Bracket stage.
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:114]: not capitalised elsewhere on this page

```suggestion
6. Only the top 32 seeded players will advance to the bracket stage.
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+     - Hidden: 2 beatmaps
+     - Hard Rock: 2 beatmaps
+     - Double Time: 3 beatmaps
+    - The draft order will be as follows:
+      1. The first player (A) will ban one beatmap, followed by the second player (B) also banning a beatmap: AB
+      2. Players will pick two beatmaps respecting the following order: BAAB
+      3. Both players ban will two maps, as such: ABBA
+      4. The last beatmap remaining in the pool will be used as the 5th pick for the match.
+   - For the second weekend, The 8 remaining players from the previous phase will then then play another set of matches, after which the remaining 4 players will be placed on a double-elimination bracket.
+     - All matches in this phase will utilise the same mappool, except for the Losers bracket Finals and Grand Finals, where a new osu! original will be introduced into the pool. The first player to score 4 points wins (best of 7).
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:134]: probably like so? i've generally seen two more common ways of writing it and i'm pretty sure both include the apostrophe. going by https://en.wikipedia.org/wiki/Double-elimination_tournament

```suggestion
     - All matches in this phase will utilise the same mappool, except for the Losers' Bracket Finals and Grand Finals, where a new [osu! original](/wiki/osu!_originals) will be introduced into the pool. The first player to score 4 points wins (best of 7).
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+       - Hidden: 3 beatmaps
+       - Hard Rock: 3 beatmaps
+       - Double Time: 4 beatmaps
+     - The draft order will be as follows:
+       1. The first player (A) will ban one beatmap, followed by the second player (B) also banning a beatmap: AB
+       2. Players will pick two beatmaps respecting the following order: BAAB
+       3. Both players will ban two beatmaps, as such: ABBA
+       4. Both players will pick one beatmap: AB
+       5. Both players will ban one beatmap: BA
+       6. The last beatmap remaining in the pool will be used as the 7th pick for the match
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:146]:
```suggestion
       6. The last beatmap remaining in the pool will be used as the 7th pick for the match.
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+       7. Exceptionally, for the Losers bracket Finals and Grand Finals, steps 5 and 6 will not be applied, and the last pick will be an osu! original beatmap, to be released at match time.
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:147]:
```suggestion
       7. Exceptionally, for the Losers' Bracket Finals and Grand Finals, steps 5 and 6 will not be applied, and the last pick will be an osu! original beatmap, to be released at match time.
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+
+### Win conditions
+
+- In the Qualifiers, players need to place in the top 32 in order to advance to the single-elimination bracket stage.
+  - Seeding for this stage will follow the %MAX format — the highest score for each map will receive 100% of the points (i.e. 1 point), and every other player will be awarded a percentage of that top score. The sum of the highest score for every map will be used as the final seeding, with a higher point sum meaning higher placement (i.e. sorted in descending order).
+    - The exact formula that will be used for all players (and for each map) is `Final Score = Player Map Score / MAX Score`, where:
+      - `Final Score` is the numerical value awarded to the current player for each map.
+      - `Player Map Score` is the score the player being evaluated achieved on the current map, the best score out of two is taken.
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:155]:
```suggestion
      - `Player Map Score` is the score the player being evaluated achieved on the current map, taking the best score out of two.
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 14]
```diff
@@ -0,0 +1,185 @@
...
+      - `MAX Score` is the highest score achieved for the current map across all players and all runs.
+- In the Round of 32 and Round of 16, players need to win 3 maps to win a match (best of 5).
+- In the Quarterfinals, Semifinals, Finals, and Grand Finals, players need to win 4 maps to win a match (best of 7).
+  - *Note: for the Grand Finals match, the player that is coming from the winners bracket is required to win one "set", that is, one best of 7, to win the tournament. The player coming from the losers bracket, however, needs to win 2 sets.*
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:159]:
```suggestion
  - *Note: for the Grand Finals match, the player that is coming from the winners' bracket is required to win one "set", that is, one best of 7, to win the tournament. The player coming from the losers' bracket, however, needs to win 2 sets.*
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 15]
```diff
@@ -0,0 +1,185 @@
...
+
+### Match procedures
+
+1. A referee will create a multiplayer room 10 minutes before the scheduled match time. players must join the lobby in that period.
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:163]:
```suggestion
1. A referee will create a multiplayer room 10 minutes before the scheduled match time. Players must join the lobby in that period.
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 16]
```diff
@@ -0,0 +1,185 @@
...
+   - For the first occurrence:
+     - The player will receive a verbal warning from the referee; in addition, the referee may opt to forcefully use the player's tactical timeout to allow for the player to pick.
+     - If the player does not pick a map within 15 seconds from when the timer expires, the referee will apply the procedure for subsequent occurrences listed below, i.e. a random pick will be used for a pick timer, and the match will be forcefully started for a ready timer.
+   - On subsequent occurrences:
+     - For a pick timer: a random map will be chosen from the mappool by rolling a random number, based on the amount of beatmaps that were neither picked nor banned.
+     - For a ready timer: the referee will issue the start command on a 10 second countdown. In this scenario, if the referee starts the match and either player is not present in the lobby, their opponent will receive the point instead.
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:172]:
```suggestion
     - For a ready timer: the referee will issue the start command on a 10-second countdown. In this scenario, if the referee starts the match and either player is not present in the lobby, their opponent will receive the point instead.
```

File: wiki/Tournaments/LGA/2024/en.md [hunk 17]
```diff
@@ -0,0 +1,185 @@
...
+     - Repeat offenders may receive further sanctions from the tournament managers.
+
+### Scheduling information
+
+1. Each stage (as described on the [Tournament schedule](#tournament-schedule) table) will be held on **a single weekend**.
+2. Matches in the Qualifiers will be held during specific time slots proposed by the tournament managers. Each player may choose a time slot in which they want to play.
+   - **Players that do not pick a time slot by Thursday, 12 July 2024 (23:59 UTC) will be disqualified from the competition.**
```
Comment (Walavouchey) [inline wiki/Tournaments/LGA/2024/en.md:179]:
```suggestion
   - **Players who do not pick a time slot by Thursday, 12 July 2024 (23:59 UTC) will be disqualified from the competition.**
```

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — DISMISSED** [review]

File: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-06-18-lga-announcement.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+This brand-new event will pit the world's 32 best players against each other in 1v1 matches played on **osu!(lazer)!**
+
+The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full Featured Artist lineup, the players will now battle each other using **cumulative score difference**! This means that if you win by 20k, you are up by 20k for the rest of the match.
+
+After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```
Comment (Albionthegreat) [inline news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md:23]:
```suggestion
After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to try out some new exciting ideas for players and viewers alike!
```
Seems to be a small typo?

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**